src/ducks/balance/components/AccountRow.jsx

```diff
@@ -91,8 +91,7 @@ class AccountRow extends React.PureComponent {
               styles['AccountRow__column--secondary']
             )}
           >
-            N°
-            {account.number}
+            {account.number && `N°${account.number}`}
```

Comment:
You can check `account.number` [here](https://github.com/cozy/cozy-banks/pull/931/files#diff-1eb84ac8f90830838bf8662f521b2539R87) to not print empty div no ?